conf/defaults.ini

```diff
@@ -773,3 +773,17 @@ license_path =
 [feature_toggles]
 # enable features, separated by spaces
 enable =
+
+[preferences]
+# Default date format
+date_format = YYYY-MM-DD HH:mm:ss
+# Used by graph and other places where we only show small intervals
+date_format_interval_PT1S = HH:mm:ss
+date_format_interval_PT1M = HH:mm
+date_format_interval_PT1H = MM-DD HH:mm
+date_format_interval_PT1D = YYYY-MM-DD
+date_format_interval_P1YT = YYYY-MM
+# Experimental feature
+date_format_use_browser_locale = false
```

Comment:
Should this be in `feature_toggles` instead?

Reply:
this is not a feature toggle but an option, not sure we every turn this on to true by default, depends on how well it works and what users prefer

Reply:
Oh right :+1: